### PR TITLE
Adding Bahasa Indonesia translation for cluster networking page

### PR DESCRIPTION
**Pull request by yofriadi**
addresses #15341

/language id

### CODE REVIEW

**Comment by girikuncoro** [comment]
@yofriadi is this a WIP or did you accidentally raise an empty PR? please raise PR when you are ready to avoid confusion or add WIP tag in the title

**Comment by yofriadi** [comment]
@girikuncoro oh sorry, this is a WIP, will update it soon, thank you.

**Comment by irvifa** [comment]
@yofriadi Can you please add a WIP in the title? Thanks..

**Comment by girikuncoro** [comment]
@yofriadi any update on this?

**Comment by yofriadi** [comment]
@girikuncoro I'm sorry, will work on it this week

**Comment by girikuncoro** [comment]
Sure no worries take your time, just checking if you are still actively
working on it :)

**Comment by irvifa** [comment]
Hi @yofriadi is this already final?

**Comment by yofriadi** [comment]
@irvifa no not yet, I guess still half way there, I will report if this done, thank you

**Comment by girikuncoro** [comment]
@yofriadi let us know when it's ready for review

**Comment by yofriadi** [comment]
@girikuncoro @irvifa Hi, I finally finish it and ready for review, sorry for being so looong.

**Comment by tengqm** [comment]
Please remove 'content/en/docs/concepts/cluster-administration/networking.md ' from your commit. It looks to me irrelevant to your main PR.

**Comment by yofriadi** [comment]
@tengqm done

File: content/id/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
+---
+reviewers:
```
Comment (irvifa) [inline content/id/docs/concepts/cluster-administration/networking.md]: Remove reviewers part in here

File: content/id/docs/concepts/cluster-administration/networking.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+4. Komunikasi eksternal dengan _service_: ini terdapat di [_services_](/docs/concepts/services-networking/service/).
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+Kubernetes adalah tentang berbagi mesin antar aplikasi. Pada dasarnya,
+saat berbagi mesin harus memastikan bahwa dua aplikasi tidak mencoba menggunakan
+_port_ yang sama. Mengkoordinasikan _port_ di banyak _developers_ sangat sulit
```
Comment (irvifa) [inline content/id/docs/concepts/cluster-administration/networking.md]:
```suggestion
_port_ yang sama. Mengkoordinasikan _port_ di banyak pengembang sangat sulit
```

Comment (irvifa) [inline content/id/docs/concepts/cluster-administration/networking.md]:
```suggestion
_port_ yang sama. Mengkoordinasikan _port_ di banyak pengembang sangat sulit
```

Comment (irvifa) [inline content/id/docs/concepts/cluster-administration/networking.md]:
```suggestion
_port_ yang sama. Mengkoordinasikan _port_ di banyak pengembang sangat sulit
```

**Comment by irvifa** [comment]
Hi  👋 @yofriadi can you please rebase and squash your commits into one? Thanks

**Comment by phanama** [comment]
Hi @yofriadi 👋 
Any updates on this??

**Comment by girikuncoro** [comment]
closing this since @wahyuoi has co-author this PR in #19420

**Pull request closed without merging**